Fabric.Identity.API/Models/Domain/DomainModelExtensions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using IdentityServer4.EntityFramework.Mappers;
+
+namespace Fabric.Identity.API.Models.Domain
+{
+    public static class DomainModelExtensions
```

Comment:
These should go in the Sql Server namespace too, same with the ITrackable and ISoftDelete interfaces since they are arguably store implementation specific.

Reply:
i will move them